src/views/virtualmachines/actions/components/DeleteVMModal/components/GracePeriodInput.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { FC } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import {
+  Checkbox,
+  Flex,
+  FlexItem,
+  InputGroup,
+  InputGroupText,
+  Popover,
+  StackItem,
+  TextInput,
+} from '@patternfly/react-core';
+import { HelpIcon } from '@patternfly/react-icons';
+
+import './grace-period-input.scss';
+
+const DEFAULT_GRACE_PERIOD = 180;
```

Comment:
it is a bit strange to me to hard-code the default for the seconds, isn't this default derived from somewhere?

Reply:
It's something defined in the backend. By default, the grace period is 3 minutes

Reply:
In the backend this value depends on the VM os. I think 3 minutes for linux distro and 1 hour for windows. Don't know if the backend exposes this kind of data but I don't think is critical here. Just to have something in the input at the beginning and 180 seconds seems reasonable. 
Having more logic here for this thing would not bring any value from my point of view.

Reply:
cant you take the default from the vm YAML (`terminationGracePeriodSeconds`)? - http://kubevirt.io/api-reference/main/definitions.html#_v1_virtualmachineinstancespec

Reply:
Great idea! I can get this value from `vm.spec.template.spec.terminationGracePeriodSeconds` and use `DEFAULT_GRACE_PERIOD` if this value is not set (cause it's optional)